dotCMS/src/main/java/com/dotcms/content/elasticsearch/util/RawActionResponse.java

```diff
@@ -0,0 +1,42 @@
+package com.dotcms.content.elasticsearch.util;
+
+
+import org.elasticsearch.action.ActionResponse;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.client.reactive.ClientHttpResponse;
+import org.springframework.web.reactive.function.BodyExtractor;
+import org.springframework.web.reactive.function.client.ClientResponse;
+
+class RawActionResponse extends ActionResponse {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A class which only has private constructors should be final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5113278)